### PR TITLE
Add maintainers

### PR DESCRIPTION
**Pull request by arajkumar**
Signed-off-by: Arunprasad Rajkumar <ar.arunprasad@gmail.com>

### CODE REVIEW

**Review by SuperQ — APPROVED** [review]
LGTM

It would be nice to get one person from prometheus-team on this list.

**Comment by arajkumar** [comment]
@SuperQ May I add your name?

**Comment by roidelapluie** [comment]
Add mine

**Comment by arajkumar** [comment]
@roidelapluie done!

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* Arunprasad Rajkumar <ar.arunprasad@gmail.com> @arajkumar
+* Prem Saraswat <prmsrswt@gmail.com> @onprem
+* Julien Pivotto <roidelapluie@o11y.eu> @roidelapluie
```
Comment (roidelapluie) [inline MAINTAINERS.md:3]:
```suggestion
* Julien Pivotto <roidelapluie@prometheus.io> @roidelapluie
```

**Comment by paulfantom** [comment]
@roidelapluie friendly ping for review :)

**Pull request merged**